lib/nifty_services/base_create_service.rb

```diff
@@ -42,15 +42,15 @@ def create_error_response(record)
     end
 
     def after_error_response(errors)
-      unprocessable_entity_error(errors) if errors.present?
+      unprocessable_entity_error(errors) unless errors.empty?
     end
 
     def after_execute_success_response
       success_created_response
     end
 
     def build_record
-      if record_type.present?
+      if !!record_type
```

Comment:
I think that `unless record_type.nil?`, type casting as boolean is an unnecessary overhead, dont you think?